wwwroot/inc/snmp.php

```diff
@@ -4361,6 +4379,36 @@
 		'text' => 'HP 1810-8G (J9449A), 8 Gigabit Ethernet 10/100/1000',
 		'processors' => array ('procurve-8ports-1000T'),
 	),
+	'9.1.1643' => array
+	(
+		'dict_key' => 3728,
+		'text' => 'Cisco Catalyst 3850-48T-S Switch, 48 10/100/1000',
+		'processors' => array ('cisco-smb-49-to-50-combo-1000SFP','cisco-smb-any-1000T'),
+	),
+	'9.1.571' => array
+	(
+		'dict_key' => 3731,
+		'text' => 'Cisco 871',
+		'processors' => array ('catalyst-chassis-any-100TX'),
+	),
+	'1.3.6.1.4.1.9.12.3.1.3.1239' => array
+	(
+		'dict_key' => 2332,
+		'text' => 'Cisco Nexus 3064-T Switch,  48 x 10GBase-T + 4 x QSFP+',
+		'processors' => array ('nexus-any-QSFP+','nexus-any-10000SFP+'),
+	),
+	'9.1.1644' => array
+	(
+		'dict_key' => 2650,
+		'text' => 'Cisco Catalyst Model WS-C3850-24T, 24 Gigabit Ethernet PoE, 2 �ombo 10 Gigabit SFP+, 4 Combo Gigabit SFP',
+		'processors' => array ('cisco-29-to-30-10000SFP+','cisco-25-to-28-1000SFP','3com-any-1000T'),
+	),
+	'9.1.1644' => array
+	(
+		'dict_key' => 3732,
+		'text' => 'Cisco SGE2000 24-Port Gigabit Switch, 24 RJ-45 connectors for 10BASE-T/100BASE-TX/1000BASE-T with 4 shared Gigabit SFP slots',
```

Comment:
The OID is incorrect (not sure about port numbers), but most importantly -- this device cannot be supported (closed Mantis #1325 with a comment).